axlearn/common/launch_trainer_main.py

```diff
@@ -2,6 +2,7 @@
 
 """Main function for launching the trainer."""
 
+import pathwaysutils  # pylint: disable=unused-import
```

Comment:
Can we do best-effort import of `pathwaysutils`? If the import is not found, we can throw an exception only if jax_backend == "proxy".

Reply:
Please address this comment so that users can continue to use launch_trainer_main.py without depending on pathways.

Reply:
Moved the import statement of pathwaysutils to the GKE-TPU Job using best effort import, since pathways will only be applied to TPUs on GKE via custom Jobset. 
Hopefully this makes more sense. I have a working example of the jobset creation w/ pathways under a draft [workload PR](https://github.com/apple/axlearn/pull/746).